services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -176,19 +179,15 @@ async def start_project_interactive_services(
         )
         for service_uuid, service in project_needed_services.items()
     ]
-
-    result = await logged_gather(*start_service_tasks, reraise=True)
-    log.debug("Services start result %s", result)
-    for entry in result:
+    results = await logged_gather(*start_service_tasks, reraise=True)
+    log.debug("Services start result %s", results)
+    for entry in results:
         if entry:
-            # if the status is present in the results fo the start_service
+            # if the status is present in the results for the start_service
             # it means that the API call failed
             # also it is enforced that the status is different from 200 OK
-            if "status" not in entry:
-                continue
```

Comment:
Sorry, but why was this removed?

I would guess that the API correctly returns a "status" key. I'd think you want it to fail fast if there is an issue and not just ignore it.

Reply:
not sure I understand your comment.
the code you currently have in master also does not fail but just logs an error.
I only refactored it only.